packages/boa/tools/install-python.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { run, PLATFORM, ARCH } = require('./utils');
+const fs = require('fs');
+const path = require('path');
+
+const CONDA_DOWNLOAD_PREFIX = 'https://repo.anaconda.com/miniconda';
+let CONDA_DOWNLOAD_NAME = 'Miniconda3-latest';
+
+if (PLATFORM === 'linux') {
+  CONDA_DOWNLOAD_NAME += '-Linux';
+} else if (PLATFORM === 'darwin') {
+  CONDA_DOWNLOAD_NAME += '-MacOSX';
+} else {
+  throw new TypeError(`No support for your platform ${PLATFORM}`);
+}
+
+if (ARCH === 'x64') {
+  CONDA_DOWNLOAD_NAME += '-x86_64';
+} else if (PLATFORM !== 'darwin') {
+  CONDA_DOWNLOAD_NAME += '-x86';
+}
+CONDA_DOWNLOAD_NAME = `${CONDA_DOWNLOAD_NAME}.sh`;
+
+if (!fs.existsSync(CONDA_DOWNLOAD_NAME)) {
+  run(`curl ${CONDA_DOWNLOAD_PREFIX}/${CONDA_DOWNLOAD_NAME} > ${CONDA_DOWNLOAD_NAME}`);
+}
+
+const CONDA_LOCAL_PATH = path.join(__dirname, '../.miniconda');
```

Comment:
I'm wondering if it's better to install .miniconda in process.cwd instead of under boa package. Otherwise if the user want to install some packages manually in conda env, it will be difficult to find this .miniconda directory. And in the meantime probably we can add a pipcook-cli command, for instance, pipcook install to let users install any packages later.

Reply:
Reasonable!